src/cs/lib/msquic_generated.cs

```diff
@@ -2380,294 +2384,294 @@ public byte RESERVED
+        internal const uint QUIC_PARAM_CONN_LOCAL_INTERFACE = 0x05000012;
 
         [NativeTypeName("#define QUIC_PARAM_CONN_TLS_SECRETS 0x05000013")]
-        public const uint QUIC_PARAM_CONN_TLS_SECRETS = 0x05000013;
+        internal const uint QUIC_PARAM_CONN_TLS_SECRETS = 0x05000013;
 
         [NativeTypeName("#define QUIC_PARAM_CONN_VERSION_SETTINGS 0x05000014")]
-        public const uint QUIC_PARAM_CONN_VERSION_SETTINGS = 0x05000014;
+        internal const uint QUIC_PARAM_CONN_VERSION_SETTINGS = 0x05000014;
 
         [NativeTypeName("#define QUIC_PARAM_CONN_CIBIR_ID 0x05000015")]
-        public const uint QUIC_PARAM_CONN_CIBIR_ID = 0x05000015;
+        internal const uint QUIC_PARAM_CONN_CIBIR_ID = 0x05000015;
 
         [NativeTypeName("#define QUIC_PARAM_CONN_STATISTICS_V2 0x05000016")]
-        public const uint QUIC_PARAM_CONN_STATISTICS_V2 = 0x05000016;
+        internal const uint QUIC_PARAM_CONN_STATISTICS_V2 = 0x05000016;
 
         [NativeTypeName("#define QUIC_PARAM_CONN_STATISTICS_V2_PLAT 0x05000017")]
-        public const uint QUIC_PARAM_CONN_STATISTICS_V2_PLAT = 0x05000017;
+        internal const uint QUIC_PARAM_CONN_STATISTICS_V2_PLAT = 0x05000017;
 
         [NativeTypeName("#define QUIC_PARAM_TLS_HANDSHAKE_INFO 0x06000000")]
-        public const uint QUIC_PARAM_TLS_HANDSHAKE_INFO = 0x06000000;
+        internal const uint QUIC_PARAM_TLS_HANDSHAKE_INFO = 0x06000000;
 
         [NativeTypeName("#define QUIC_PARAM_TLS_NEGOTIATED_ALPN 0x06000001")]
-        public const uint QUIC_PARAM_TLS_NEGOTIATED_ALPN = 0x06000001;
+        internal const uint QUIC_PARAM_TLS_NEGOTIATED_ALPN = 0x06000001;
 
         [NativeTypeName("#define QUIC_PARAM_TLS_SCHANNEL_CONTEXT_ATTRIBUTE_W 0x07000000")]
-        public const uint QUIC_PARAM_TLS_SCHANNEL_CONTEXT_ATTRIBUTE_W = 0x07000000;
+        internal const uint QUIC_PARAM_TLS_SCHANNEL_CONTEXT_ATTRIBUTE_W = 0x07000000;
 
         [NativeTypeName("#define QUIC_PARAM_STREAM_ID 0x08000000")]
-        public const uint QUIC_PARAM_STREAM_ID = 0x08000000;
+        internal const uint QUIC_PARAM_STREAM_ID = 0x08000000;
 
         [NativeTypeName("#define QUIC_PARAM_STREAM_0RTT_LENGTH 0x08000001")]
-        public const uint QUIC_PARAM_STREAM_0RTT_LENGTH = 0x08000001;
+        internal const uint QUIC_PARAM_STREAM_0RTT_LENGTH = 0x08000001;
 
         [NativeTypeName("#define QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE 0x08000002")]
-        public const uint QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE = 0x08000002;
+        internal const uint QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE = 0x08000002;
 
         [NativeTypeName("#define QUIC_PARAM_STREAM_PRIORITY 0x08000003")]
-        public const uint QUIC_PARAM_STREAM_PRIORITY = 0x08000003;
+        internal const uint QUIC_PARAM_STREAM_PRIORITY = 0x08000003;
 
         [NativeTypeName("#define QUIC_API_VERSION_2 2")]
-        public const uint QUIC_API_VERSION_2 = 2;
+        internal const uint QUIC_API_VERSION_2 = 2;
     }
 }
```

Comment:
```suggestion
}
#pragma warning restore CS0649
```

Reply:
Thats actually difficult to do in the generator. Is it needed to restore, even though its a generated file?

Reply:
Not really, it should get restored at the end of the file anyway.